google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriter.java

```diff
@@ -228,7 +228,7 @@ public static final class Builder {
 
     private Builder(SchemaAwareStreamWriter.Builder<Object> schemaAwareStreamWriterBuilder) {
       this.schemaAwareStreamWriterBuilder =
-          schemaAwareStreamWriterBuilder.setTraceIdBase("JsonWriter");
+          schemaAwareStreamWriterBuilder.setClientId("java-jsonwriter");
```

Comment:
change to static final

Reply:
Do you mean this? In theory this can be set too but we just don't bother exposing it.